source/includes/language-compatibility-table-cxx.rst

```diff
@@ -8,24 +8,14 @@
      - C++17
      - C++14
      - C++11
-     - C++03
 
-   * - 3.8+
+   * - 3.8 to 3.10
      - ✓
      - ✓
      - ✓
      - ✓
-     -
-
-   * - <= 3.7
-     -
-     - ✓
-     - ✓
-     - ✓
-     -
 
-   * - 1.x
```

Comment:
Tech reviewer: I couldn't find a v1.x driver in either the docs or driver repo. If it needs to be added, let me know.

Reply:
Versions older than 3.0 correspond to the legacy driver. The legacy driver is [documented as end-of-life](https://mongocxx.org/legacy-v1/). I expect the legacy driver does not need to be mentioned.